scripts/prepare-release

```diff
@@ -78,8 +78,8 @@ if [[ true ]]; then
   new_version_name="Tenzir ${new_version_major_minor}"
   jq "map(select(. | startswith(\"${new_version_name%-*}\") | not))" < versions.json > versions.json.tmp
   mv -f versions.json.tmp versions.json
-  rm -rf "versioned_docs/version-${new_version_name%-*}"*
-  rm -rf "versioned_sidebars/version-${new_version_name%-*}"*-sidebars.json
+  rm -rf "versioned_docs/version-${new_version_name%-*}"* || true
+  rm -rf "versioned_sidebars/version-${new_version_name%-*}"*-sidebars.json || true
```

Comment:
FYI: This modification is a follow up to #3278. It is a workaround for idiosyncratic behavior of zsh, which fails on a globbed `rm` if no matches are found.

Reply:
This should be a comment in the code.

Reply:
I'll follow that up with a separate PR. The script should add a PR description as well.